examples/TypeScriptMessaging/index.js

```diff
@@ -1,7 +1,9 @@
 import 'react-native-gesture-handler';
-import { AppRegistry } from 'react-native';
+import { AppRegistry, I18nManager } from 'react-native';
 
 import App from './App';
 import { name as appName } from './app.json';
 
+I18nManager.forceRTL(true);
```

Comment:
Why do we still have it here @Enigma-I-am ?

Reply:
I've resolved it